ssl-config/src/main/scala/com/avast/sst/ssl/SslContextModule.scala

```diff
@@ -0,0 +1,26 @@
+package com.avast.sst.ssl
```

Comment:
the directory should be named `ssl-context` (not `ssl-config`)

Reply:
No, it's based on the dependency which is named `ssl-config`. If anything I could rename the package to `sslconfig` or something like that but I though just `ssl` would work better.

Reply:
:/ sorry, I'm probably confused... the point of this (sub)project is to create an instance of `javax.net.ssl.SSLContext`, no? that's why I would expect the name to be `ssl-context`

Reply:
Yes but the names of the modules are based on the dependencies, not the contens of the module (though these are very related usually ;) ). Theoretically there could be multiple implementations of how to create `SSLContext` so if I name it just `sst-ssl-context` I wouldn't know which dependency/implementation it is and if I wanted to have both in this project I would have a conflict. I understand your point but this way it is extensible and consistent with the rest of the modules.

Reply:
you're absolutely on point that this is just one way to create a `SSLContext`, and there could be many other, at least in theory. Maybe the subproject name should reflect that: `sslcontext-typesafe`?

Reply:
The subproject reflects that because the library is called `ssl-config`: https://github.com/lightbend/ssl-config